apps/trading/components/order-list-container/order-list-container.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import '@testing-library/jest-dom';
```

Comment:
chore (blocking): Don't like having to import this is every test file.

Can you add a setupsTests.ts file and call it from the jest config using the `setupAfterEnv` config property.

I think @sam-keen has a PR that does exactly this in #85.

I think there is another package that needs this as well